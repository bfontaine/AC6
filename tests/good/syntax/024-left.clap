val x : { Left + Right } = Left at { Left + Right }

