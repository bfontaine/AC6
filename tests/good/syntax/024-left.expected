val x : { Left + Right } =
  Left { Left + Right }
